algorithms/CPlusPlus/Graphs/topological-sort.cpp

```diff
@@ -0,0 +1,135 @@
+//Topological sort to check order using dfs
+
```

Comment:
Use [c++ code beautify](https://codebeautify.org/cpp-formatter-beautifier) to fix indent and extra spaces

Reply:
Can you check the changes